src/components/Articles.jsx

```diff
@@ -29,10 +29,12 @@ class Articles extends Component {
 			rows = Object.keys(this.props.articles.data).map((qid) => {
 				const article = this.props.articles.data[qid];
 				const status = this.props.articles.status[qid];
+				/* eslint-disable react/jsx-no-bind */
```

Comment:
Instead of disabling the rule, it would be better to call the methods from the Article component, as explained here: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#protips